### PR TITLE
Mount path change for config

### PR DESCRIPTION
**Pull request by jeffbanks**
Support reaper image permissions being applied in k8ssandra security context configurations.

* The Reaper config file is stored at /etc/cassandra-reaper.yml
* Moving to /etc/reaper/cassandra-reaper.yml
* [k8ssand-701](https://k8ssandra.atlassian.net/browse/K8SSAND-701)

### CODE REVIEW

**Review by adejanovski — DISMISSED** [review]
I successfully tested these changes on the Docker image ✅ 

Feel free to squash and merge the PR so we only get a single commit in `master`.

**Comment by jeffbanks** [comment]
@jsanda @adejanovski 
I'm adding a link to the [reaper-operator PR](https://github.com/k8ssandra/reaper-operator/pull/68) as a reference that will utilize this change.

File: src/server/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -97,26 +97,26 @@ ADD ${SHADED_JAR} /usr/local/lib/cassandra-reaper.jar
...
 
 
-# get around `/usr/local/bin/configure-persistence.sh: line 65: can't create /etc/cassandra-reaper.yml: Interrupted system call` unknown error
-RUN touch /etc/cassandra-reaper.yml
-# get around `/usr/local/bin/configure-webui-authentication.sh: line 44: can't `create` /etc/shiro.ini: Interrupted system call` unknown error
-RUN touch /etc/shiro.ini
+# get around `/usr/local/bin/configure-persistence.sh: line 65: can't create /etc/reaper/cassandra-reaper.yml: Interrupted system call` unknown error
+RUN touch /etc/reaper/cassandra-reaper.yml
+# get around `/usr/local/bin/configure-webui-authentication.sh: line 44: can't `create` /etc/reaper/shiro.ini: Interrupted system call` unknown error
+RUN touch /etc/reaper/shiro.ini
```
Comment (adejanovski) [inline src/server/src/main/docker/Dockerfile:103]: This looks inconsistent with line 108 and 122 where the `shiro` directory is created under `/etc/cassandra-reaper`

**Review by adejanovski — CHANGES_REQUESTED** [review]
We seem to be creating both `/etc/cassandra-reaper` and `/etc/reaper` here.
I guess `/etc/cassandra-reaper` would be more consistent with how we deploy Reaper in non containerized environments.
If you go with `/etc/reaper`, could you change the `/etc/cassandra-reaper` occurences to it?
Thanks

File: src/server/src/main/docker/Dockerfile (hunk 1 above)
Comment (jeffbanks) [reply src/server/src/main/docker/Dockerfile:103]: Good find.  There were 3 locations related to shiro here that I'll update.

**Review by adejanovski — APPROVED** [review]
LGTM now. Thanks @jeffbanks!
Feel free to merge this PR.

**Pull request merged**